src/offset/local/windows.rs

```diff
@@ -119,3 +119,82 @@ fn systemtime_to_naive_dt(st: SYSTEMTIME, year: i32) -> Option<NaiveDateTime> {
     };
     Some(date.and_time(time))
 }
+
+#[test]
```

Comment:
I'd like to have this squashed into the previous commit so that I can compare the test and the previous code more easily.